### PR TITLE
feat(ci): create issue template

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

Since we will allow users to give feedback, we will try and standardize the issue reports in Github for this project. However, we leave the possibility to create custom issues.

### CODE REVIEW

**Review by anedos-dfinity — APPROVED** [review]
I like it, thanks.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+blank_issues_enabled: true  # Allows users to open a blank issue.
+
+# You can also add external links or contact information here
+contact_links:
+  - name: Support
+    url: TBD
```
Comment (bitdivine) [inline .github/ISSUE_TEMPLATE/config.yml:6]: Forum maybe?  What are the options?

**Review by bitdivine — APPROVED** [review]
LGTM, Thanks!

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply .github/ISSUE_TEMPLATE/config.yml]: for now we remove it then

**Pull request merged**